.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+[ ] I am you using the latest minor version of the major version (x.y.last)?
```

Comment:
Perhaps can we add:

\+ [ ] I've assigned some labels at this Issue
\+ [ ] I've assigned a milestone at this issue